### PR TITLE
Update Central upgrade instruction structure to make commands copiable + draw attention to relevant version upgrade notes

### PR DESCRIPTION
**Pull request by lognaturel**
Goals:
- Make it easy to copy shell commands
- Improve the experience for people who upgrade every time
- Make it clear that instructions for every version between the current and target upgrade version are relevant
- Match https://docs.getodk.org/central-troubleshooting/#database-reset-after-running-docker-command which @yanokwa recently restructured

Todo:
- Add general instructions for performing maintenance: make a backup, announce a maintenance period, etc

@yanokwa, @alxndrsn, I'm interested in your early thoughts. I think this structure will make it easier to make the postgres upgrade instructions visible.

### CODE REVIEW

**Comment by alxndrsn** [comment]
The scope of this PR seems beyond what's described.  Some of the changes look 100% acceptable and could be merged immediately.

Making instructions copyable sounds like a great idea!

**Comment by lognaturel** [comment]
@matthew-white this intersects with docs you're writing so you may want to take a quick look to see whether you like the proposed structure.

File: docs/central-upgrade.rst [hunk 1]
```diff
@@ -3,52 +3,90 @@
...
 =================
 
-We release new versions of Central regularly. You do not have to upgrade to the latest version immediately, but we generally recommend that you do so to get access to the newest features, bug fixes, and security updates.
+We release new versions of Central regularly. We generally recommend that you upgrade as soon as you can to get access to the newest features, bug fixes, and security updates. It is not generally possible to downgrade and we do not guarantee that versions prior to the latest release can still be installed.
 
-.. admonition:: Note
+Start by reviewing upgrade notes for all versions between your current version and the one you are upgrading to.
 
-  To see your version number, click on the question mark icon in the upper right section of your Central menu bar, then click :guilabel:`Version`. If you don't see the question mark, you can also see the version number by adding ``version.txt`` to the root URL (e.g., `demo.getodk.cloud/version.txt <https://demo.getodk.cloud/version.txt>`_).
+* Central v1.5: no special instructions.
```
Comment (matthew-white) [inline docs/central-upgrade.rst:10]: Why mention v1.5 in particular?

**Review by matthew-white — COMMENTED** [review]
The new structure looks great to me! I like that we're making it very clear when there are version-specific instructions. Also, I think this PR closes #1381.

File: docs/central-upgrade.rst (hunk 1 above)
Comment (yanokwa) [reply docs/central-upgrade.rst:10]: People on the forum sometimes ask if there are upgrade instructions for a version if we don't explicitly document that there are none.

Comment (matthew-white) [reply docs/central-upgrade.rst:10]: That makes sense to me. I guess I'm wondering why we'd mention v1.5 and not other versions, for example, v2022.3 or v1.1. Could we instead add a generic note along the lines of "Most versions of Central do not have special upgrade instructions"?

Comment (matthew-white) [reply docs/central-upgrade.rst:10]: (Also, definitely don't want to block merging / no strong feelings, mostly just wanted to throw out the thought.)

**Comment by lognaturel** [comment]
Thanks, all. Don't hesitate to make edits in follow-on PRs.

**Pull request merged**